static/app/views/dashboardsV2/widgetCard.tsx

```diff
@@ -53,7 +53,7 @@ type Props = WithRouterProps & {
   noLazyLoad?: boolean;
 };
 
-class WidgetCard extends React.Component<Props> {
+export class WidgetCard extends React.Component<Props> {
```

Comment:
Why export this? Can't use use the one with all the HoCs?

Reply:
Yes, I was getting a `Type 'X' is not a constructor function type` issue, but since I'm not depending on this component anymore it's a non-issue!